Docs/developer_guide/module_overview.md

```diff
@@ -73,7 +73,7 @@ Recommended for fast prototyping and custom workflow development.
 Specifications:
 * Written in Python.
 * Full [Slicer API](api.md#python) is accessible.
-* Full access to the API of [MRML](mrml.md), [VTK](http://www.vtk.org), [Qt](https://doc.qt.io/), [ITK](https://itkpythonpackage.readthedocs.io) and [SimpleITK](http://www.itk.org/SimpleITKDoxygen/html/classes.html) since are Python wrapped.
+* Full access to the API of [MRML](mrml.md), [VTK](http://www.vtk.org), [Qt](https://doc.qt.io/), [ITK](https://itkpythonpackage.readthedocs.io) and [SimpleITK](https://simpleitk.readthedocs.io/en/master/) since are Python wrapped.
```

Comment:
```suggestion
* Full access to the API of [MRML](mrml.md), [VTK](https://docs.vtk.org), [Qt](https://doc.qt.io/), [ITK](https://docs.itk.org/) and [SimpleITK](https://simpleitk.readthedocs.io/) since those are Python wrapped.
```